src/odemis/gui/cont/tabs.py

```diff
@@ -3243,9 +3233,6 @@ def __init__(self, name, button, panel, main_frame, main_data):
             self.panel.pnl_streak.Show(False)
         else:
             self.panel.btn_manualfocus.Show(True)  # TODO only for streak cam now
-            # Bind event
-            self.panel.btn_open_streak_calib_file.Bind(wx.EVT_BUTTON, self._settings_controller._onOpenCalibFile)
-            self.panel.btn_save_streak_calib_file.Bind(wx.EVT_BUTTON, self._settings_controller._onSaveCalibFile)
```

Comment:
You still need to bind the buttons in the SettingsController class... or did I miss some magic?